collaboration/connectionmanager.py

```diff
@@ -16,28 +16,35 @@
 # Boston, MA 02111-1307, USA.
 
 """
-UNSTABLE. It should really be internal to the sugar.presence package.
+UNSTABLE. It should really be internal to the sugar3.presence package.
 """
 
 from functools import partial
 
 import dbus
 from dbus import PROPERTIES_IFACE
-from telepathy.interfaces import (ACCOUNT, ACCOUNT_MANAGER)
-from telepathy.constants import CONNECTION_STATUS_CONNECTED
+
+from gi.repository import TelepathyGLib
+
+# from telepathy.interfaces import (ACCOUNT, ACCOUNT_MANAGER)
+ACCOUNT = TelepathyGLib.IFACE_ACCOUNT
+ACCOUNT_MANAGER = TelepathyGLib.IFACE_ACCOUNT_MANAGER
+CONNECTION_STATUS_CONNECTED = TelepathyGLib.ConnectionStatus.CONNECTED
```

Comment:
If this isn't used, there might be no need to add them.

Reply:
We may also need to update upstream TurtleArt, I have rebased it with TutrleArt.

Reply:
These constants are used in file later. I dont think it is good to remove now.

Reply:
If they're used then there's no reason to remove them.